compose_generator/odcs_fetcher.py

```diff
@@ -11,14 +14,37 @@ class ODCSResultReference(ComposeReference):
     Reference to a locally-stored compose result
     """
 
-    compose_path: Path
+    compose_file_path: Path
 
 
 @dataclass(frozen=True)
 class ODCSFetcher(ComposeFetcher):
     """
     Fetch ODCS compose based on a remote compose-reference and store it locally
+
+    :param compose_file_path: The Desired path for the compose file.
+                              must include the full path to the file as
```

Comment:
i think relative path should be good as well.
```suggestion
                              must include the path to the file as
```